src/RequestHandler.php

```diff
@@ -261,7 +261,7 @@ public function slaveConnectFailed(\Exception $e)
     protected function verboseTimer($callback, $always = false)
     {
         $took = microtime(true) - $this->start;
-        if ($this->output->isVeryVerbose() && ($always || $took > 1)) {
+        if (($always || $took > 1) && $this->output->isVeryVerbose()) {
```

Comment:
Why- performance?

Reply:
Kind of: lazy conditions.

Reply:
It makes a lazy function call. I feel (and nothing more) that it slightly impacts readability.

Reply:
You are the boss =) Reverting?

Reply:
Your way- saves the call